MH4U Dex/CoreDataController.m

```diff
@@ -0,0 +1,430 @@
+//
+//  CoreDataController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 2/6/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "CoreDataController.h"
+#import <NSManagedObjectContext+Hydrate.h>
+#import "MonsterDrop.h"
+#import "AreaDrop.h"
+#import "Item.h"
+#import "Monster.h"
+#import "Region.h"
+#import "Area.h"
+
+@implementation CoreDataController
+
+#pragma mark - Initial Data Loading
+
+- (void)loadMonsterData
+{
+    // Because monsters are added first, there are no relationships to set up, so we can hydrate them.
+    NSString *path = [[NSBundle mainBundle] pathForResource:@"monsters" ofType:@"json"];
+    NSDictionary *attributes = @{@"id":@"_id", @"monster_class":@"class", @"name":@"name", @"jpn_name":@"jpn_name", @"sort_name":@"sort_name", @"trait":@"trait", @"icon":@"icon_name", @"url":@"url"};
+    [self.managedObjectContext hydrateStoreWithJSONAtPath:path attributeMappings:attributes forEntityName:@"Monster"];
+    path = [[NSBundle mainBundle] pathForResource:@"damageZones" ofType:@"json"];
+    attributes = @{@"id":@"_id", @"monster_id":@"monster_id", @"monsterName":@"monster_name", @"jpnMonsterName":@"jpn_monster_name", @"bodyPart":@"body_part", @"cut":@"cut", @"impact":@"impact", @"shot":@"shot", @"fire":@"fire", @"water":@"water", @"thunder":@"thunder", @"ice":@"ice", @"dragon":@"dragon", @"ko":@"ko", @"extract":@"extract"};
+    [self.managedObjectContext hydrateStoreWithJSONAtPath:path attributeMappings:attributes forEntityName:@"DamageZone"];
```

Comment:
With mogenerator, you can call the `entityName` class method on the corresponding class rather than hard-coding the string for the entity name.

Does "Damage Zone" loading conceptually belong inside `loadMonsterData`?